.travis.yml

```diff
@@ -24,7 +24,7 @@ install:
 
 script:
   - flake8 aiohttp
-  - if python -c "import sys; sys.exit(sys.version_info >= (3,5)"; then
+  - if python -c "import sys; sys.exit(sys.version_info >= (3,5))"; then
```

Comment:
Heh, I believe it worked "as expected" with that typo)
But real check must look like this:
```yaml
- if python -c "import sys; sys.exit(sys.version_info < (3,5))"; then
      flake8 examples tests;
  fi
```
`sys.exit()` must be called with `0 / False` for non-errored runs.